src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -2907,7 +2907,7 @@ def _process_vnet_name_and_id(vnet):
                 name=vnet)
         return vnet
     if backend_addresses and backend_addresses_config_file:
-        raise CLIError('usage error: --backend-address [--backend-address] | --backend-addresses-config-file')
+        raise CLIError('usage error: --backend-address [--backend-address] [--vnet] | --backend-addresses-config-file [--vnet]')
```

Comment:
>| [](start = 84, length = 1)

usage error: Only one of --backend-address and --backend-addresses-config-file can be provided at the same time.